src/test/java/org/testng/GuiceHelperTest.java

```diff
@@ -0,0 +1,61 @@
+package org.testng;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertTrue;
+
+import org.testng.annotations.Guice;
+import org.testng.annotations.Test;
+import org.testng.internal.ClassImpl;
+import org.testng.internal.paramhandler.FakeTestContext;
+
+import com.google.inject.Injector;
+import com.google.inject.Module;
+import com.google.inject.Stage;
+
+import test.guice.FakeInjector;
+
+/**
+ * @author Bartosz Popiela
+ */
+@Guice
+public final class GuiceHelperTest {
```

Comment:
Can we please move all the guice related tests to `test.guice` package ?

Reply:
GuiceHelper#getInjector is package-private.